library-best-practices/JLBP-12.md

```diff
@@ -0,0 +1,8 @@
+[JLBP-12] Make level of support and level of API stability clear
+----------------------------------------------------------------
+
+- Surface stability should be clearly described.
+- Feedback mechanisms should be outlined (for example, file an issue).
```

Comment:
Provide and identify a feedback mechanism. For example,

* File an issue at https://github.com/...
* Send email to dev@fooproject.example.com